### PR TITLE
Adding an image so I can refer to it without breaking the build.

### PR DESCRIPTION
**Pull request by sfshaza2**
Adding an image that shows the anatomy of an app as created by "flutter create".

### CODE REVIEW

**Review by atsansone — APPROVED** [review]
I would suggest including what's on the left inside box with bigger type, but that doesn't mean this can't merge as is. Your call.

**Comment by sfshaza2** [comment]
Yeah, to do that, I'd have to pester the engineer who created it.

**Pull request merged**
